SwifterSwift.xcodeproj/project.pbxproj

```diff
@@ -1309,6 +1314,14 @@
 			path = HealthKitTests;
 			sourceTree = "<group>";
 		};
+		587AC78525C1D4C7009977B5 /* Deprecated */ = {
+			isa = PBXGroup;
+			children = (
+				587AC78625C1D4F0009977B5 /* UIKitDeprecated.swift */,
```

Comment:
We put a `Deprecated` folder inside the parent folder, in this case `Sources/SwifterSwift/UIKit`. You can see that one already exists in `SwiftStdlib`. This is required because if somebody decides to use only a subspec then the deprecated folder will need to be included there.

Reply:
Yeah, this `Deprecated` folder located at `Sources/SwifterSwift/UIKit`, and I moved the deprecated functions and enum in a file named `UIKitDeprecated.swift` in this folder.

Did I miss something?

Reply:
Ah, you did it correctly. I was just reading from the PR and it looked like you'd placed the folder at the top level. I can see that it's correct :D